tests/Middleware/updateToppingValidator.test.js

```diff
@@ -0,0 +1,13 @@
+const checkInt = require("../../Middleware/updateToppingValidator").checkInt;
+
+test("input as a string", () => {
+  expect(checkInt("342")).toBe(false);
+});
+
+test("input with decimal", () => {
+  expect(checkInt(34.24)).toBe(false);
+});
+
+test("input with decimal", () => {
```

Comment:
This aint a decimal!

Reply:
oh dear, if Maria went to Oxford she may have realised this!